docs/blog/Webots-2022-a-release.md

```diff
@@ -0,0 +1,101 @@
+
+**ENU** means _East_ along the _X_-positive axis, _North_ along the _Y_-positive axis and _Up_ along the _Z_-positive axis
+
+
+In addition, **the object's axis system of Webots is now FLU** (_x-**F**orward, y-**L**eft, z-**U**p_). Whereas before there was no real preferred object's axis system.
+
+%figure "On the left, Webots R2021b, on the right Webots R2022a"
+![Axis illustration](images/flu-enu.png)
+%end
+
+### Why These Changes?
+
+ENU and FLU are the most widely used axis conventions in robotics, including ROS and ROS2.
+
+More and more people are using Webots with ROS2 and we changed the axis convention of Webots to make it easier for them !
+
+### What Changed?
+
+We worked hard to convert all PROTOS and worlds distributed with Webots for you, so that they are ready to use in R2022a.
+
+A mechanism of backward compatibility has been implemented directly in Webots.
+Unfortunately, it was not possible to make it work in every cases.
+It will try to convert worlds but cannot convert protos.
+
+We also wrote a [guide](https://github.com/cyberbotics/webots/wiki/FLU-and-ENU-conversion-guide) to help you convert your own protos and worlds to use them in Webots R2022a.
+
+We are aware that it is a major change and it can be annoying to convert your worlds and PROTOS to the new format.
+However, it is a worthwhile effort, especially if you work with ROS.
+
+---
+
+## Skin Node
+
+A new node, the [Skin](../reference/skin.md) node has been introduced to simulate soft mesh animation.
+This is particularly useful to simulate humans or animals.
+
+%figure "Skin examples"
+![Skin examples](images/skin.png)
+%end
+
+---
+
+## Improving User Interface
+
+Several improvement enhanced the user experience:
+
+  - Rotating objects directly in the scene is now easier than before.
+  - The zooming speed has been adjusted to make the navigation more pleasant.
+  - Webots now displays a warning when a user attempts to add a node in an already started simulation.
+
+---
+
+## ROS and ROS 2
+
+The Webots ROS and ROS 2 interfaces were maintained and updated to benefit from the new features of Webots R2022a, such as the ENU/FLU orientations, the Skin node and the GPS speed vector.
+New versions of the `webots_ros` package were released for [melodic](https://index.ros.org/p/webots_ros/#melodic) and [noetic](https://index.ros.org/p/webots_ros/#noetic).
+
+In addition to this, the Webots ROS 2 interface was fully redesigned to be more user friendly and scalable.
+The `webots_ros2_driver` sub-package replaces the previous `webots_ros2_core` sub-package, which is now obsolete.
+This new sub-package automatically creates a ROS 2 interface out of a Webots robot model, allowing users to configure the ROS 2 interface in URDF files.
+Moreover, users can extend the interface using the [pluginlib](https://github.com/ros/pluginlib/tree/ros2) plugin mechanism.
```

Comment:
```suggestion
Moreover, users can extend the interface by creating C++ (with [pluginlib](https://github.com/ros/pluginlib/tree/ros2) plugin mechanism) or Python plugins.
```
Only C++ plugins are based on pluginlib, so it might be more appealing to let users know they can use C++ and Python create plugins.